tracer/test/snapshots/Rasp.PathTraversal.AspNetCore2.IastDisabled.verified.txt

```diff
@@ -0,0 +1,51 @@
+﻿[
```

Comment:
Since we still have not implemented any RASP vulnerability, we just check that having enabled RASP with or without IAST doesn't change previous behavior.